docs/DG_Diagrams/RouteCommandClass.puml

```diff
@@ -0,0 +1,51 @@
+@startuml
+
+abstract class Command
+Class RouteCommand
+Class RouteParser
+Class Parser
+Class BusData {
+{static} possibleBuses (:String, :String) : ArrayList<Bus>
+}
+Class Bus {
+getPossibleRoute(:String, :String) : ArrayList<BusStops>
+}
+Class SimilarityCheck {
+{static} similarLoc(:String) : ArrayList<String>
+}
+
+Class Ui {
+{static} printRouteMessage(:ArrayList<Bus>)
+{static} printPossibleLocsMessage(:ArrayList<String>)
+}
+
+enum BusStops {
+PGP
+KENTRIDGEMRTSTATION
+...
+COLLEGEGREEN
+OEITIONGHAM
+}
+
+enum ExceptionType {
+NO_ROUTE_DELIMITER
+NO_LOCATIONS
+SAME_LOCATIONS
+...
+}
+
+Command <|- RouteCommand
+Parser <|- RouteParser
+RouteParser "1"<--down RouteCommand
+RouteCommand .> BusData
+BusData --> "0..*" Bus
+Bus ..> BusStops
+RouteCommand "throws"..> CustomException
```

Comment:
![e4](https://user-images.githubusercontent.com/60377900/97546345-49c9fe80-1a07-11eb-8b3f-9deacb8a8cd7.png)

Is the throws for the dotted line or solid line? It's a bit unclear as it is in between the two lines